packages/connect-wallet-modal/src/components/Ledger/LedgerAccountButton.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC } from 'react';
+import { Address, Text, useBreakpoint } from '@lidofinance/lido-ui';
+import styled, { keyframes } from '@reef-knot/ui-react/styled-wrapper';
+import { LedgerWalletIcon } from './icons/LedgerWalletIcon';
+import { AccountRecord } from './types';
+
+const ButtonStyled = styled.div`
+  width: 100%;
+  background-color: ${({ theme }) => theme.colors.backgroundSecondary};
+  border-radius: 10px;
+  display: flex;
+  padding: 16px 20px;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 10px;
+
+  :first-child {
```

Comment:
I do not remember surely, but I think that this will be equivalent to `& :first-child` (any children of this element matching), not `&:first-child`, can you please double-check it?

Reply:
it produces expected css and works like `&:first-child`:
<img width="250" alt="image" src="https://github.com/lidofinance/reef-knot/assets/1245209/060c20ce-1bd0-4a74-b4ec-d1d5d788c884">

Reply:
OK, good